x/stake/keeper/query_utils.go

```diff
@@ -0,0 +1,173 @@
+package keeper
+
```

Comment:
these are defined on the keeper itself. They should probably live with the rest of the keeper methods and not in `query_utils.go`

Reply:
I see.